geometry_msgs/Vector3 loc
int32 timer
int32 persistence
float64 probability 